app/css/cohortbuilder.css

```diff
@@ -107,6 +107,28 @@ li>a:hover {
 li>a:focus {
     color: gold;
 }
+.table thead {
+    position: -webkit-sticky;
+    position: -moz-sticky;
+    position: -ms-sticky;
+    position: -o-sticky;
+    position: sticky;
+    top: 0;
+    background-color: #FFF;
+}
+.table {
+    table-layout: fixed;
+    width: 100%;
+}
+
+th, td {
+    width: 180px;
+    word-wrap:break-word;
+}
+
+.index-size {
```

Comment:
The index-size class is used to size the S/N column that consists of row indices. With this could we get a more descriptive name?